docs/graphql/manual/schema/relay-schema.rst

```diff
@@ -0,0 +1,69 @@
+.. meta::
+   :description: Using Hasura's Relay API
+   :keywords: hasura, docs, Relay, schema, API
+
+.. _relay_schema:
+
+Relay schema
+============
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+The Hasura GraphQL Engine serves a `Relay <https://relay.dev/>`__ schema for Postgres tables which have a primary key defined. The Relay schema can be accessed through the ``/v1/relay`` endpoint.
```

Comment:
We should also add this endpoint on [this page](https://deploy-preview-5150--hasura-docs.netlify.app/graphql/manual/api-reference/index.html#available-apis).